frontend/src/views/AboutView.vue

```diff
@@ -1,3 +1,214 @@
 <template>
-  <div>About</div>
+  <v-container>
+    <div id="about">
+      <h1>{{ titlePage }}</h1>
+      <h2>{{ projectName }}</h2>
+      <br />
+      <p>
+        ft_transcendence is the last project of the 42 common core. It is
+        designed to complete the first part of our cursus. We had to program, as
+        a team, something completely different from what we have done so far. We
+        were supposed to create a website where users can face each other in
+        Pong games. The website where you are now! <br />The specifications
+        included: a nice user interface, a chat, and real-time multiplayer
+        online games, compatible with the latest stable up-to-date version of
+        Google Chrome, Firefox, and Safari. <br />We have developed our website
+        with these technologies:
+      </p>
+      <ul>
+        <li>backend: NestJS;</li>
+        <li>frontend: TypeScript, with Vue.js as framework;</li>
+        <li>database: PostgreSQL.</li>
+      </ul>
+      <br />
+      <p>
+        For more information about the rules we had to follow, you can check our
+        <a
+          href="https://github.com/fyusuf-a/ft_transcendence/blob/master/README.md"
+          target="_blank"
+          >README.md</a
+        >.
+      </p>
+    </div>
+    <div id="app">
+      <h2>{{ team }}</h2>
+      <br />
+      <v-app id="inspire">
+        <v-container class="grey lighten-5">
+          <v-row no-gutters class="pb-4">
+            <v-col class="pl-2 pr-2 pt-4">
+              <v-card class="pa-2 pr-2 mx-auto" tile>
+                <v-col align="center" class="mt-5">
+                  <v-avatar size="100">
+                    <v-img class="photos" v-bind:src="tvideiraPic"></v-img>
+                  </v-avatar>
+                  <v-card-text class="text--primary">
+                    tvideira<br />name?<br />
+                    <div>
+                      <a
+                        v-bind:href="'https://github.com/tvideira'"
+                        target="_blank"
+                      >
+                        <img
+                          id="github"
+                          style="max-height: 30px"
+                          v-bind:src="githubLogo"
+                          v-bind:alt="githubAlt"
+                        />
+                      </a>
+                    </div>
+                    [other, like mail?]
+                  </v-card-text>
+                </v-col>
+              </v-card>
+            </v-col>
```

Comment:
Just a suggestion, but this could be a good opportunity to use a sub-component (and maybe a loop) to reduce code duplication

A component that defines the layout for each team member's column could take the photo path, login, and github profile url as `props` and then be reused for each team member (manually or within a loop over an array of those variables)